recipe/meta.yaml

```diff
@@ -11,115 +11,126 @@ source:
   sha256: 9db4db2133a52e48af043fbb3bb920a77886f64a6a1c8cd56adee41828487fc6
 
 build:
-  number: 1
+  number: 0
+  skip: true  # [py<38]
 
 outputs:
   - name: allure-python-commons
-    build:
-      noarch: python
     requirements:
       host:
         - pip
-        - python >=3.6
+        - python
+        - setuptools
+        - wheel
         - setuptools-scm
       run:
-        - python >=3.6
+        - python
         - attrs >=16.0.0
         - pluggy >=0.4.0
-    script: install.sh
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
```

Comment:
Could you place these lines under   `- name: allure-python-commons` and correspondingly for other outputs?

Reply:
done